lib/northbound_grpc.cpp

```diff
@@ -1403,6 +1403,25 @@ static int frr_grpc_finish(void)
 	return 0;
 }
 
+/**
+ * Calls the protobuf library shutdown function in order to clean up
+ * used memory.
+ *
+ * **NOTE**
+ * This was made into a new function to causing crashes with the existing
```

Comment:
Feel free to rewrite this function doc. I made terrible mistakes while writing it:
```
 * This was made into a new function to causing crashes with the existing
 * zebra FPM protobuf module.
```

What I meant was: I wrote a new separated shutdown function so it'd be called before the daemon exit. At that time the `frr_grpc_finish` was called in `frr_early_fini` and the protobuf library shutdown must be called as the last thing (e.g. the old FPM module could be using it during `frr_early_fini`).

Since `frr_gprc_finish` and `frr_protobuf_finish` are in the same finish hook (`frr_fini`) we can also merge them.